roles/confluent.common/tasks/failure_handling.yml

```diff
@@ -15,13 +15,37 @@
     recurse: false
   register: find_output
 
-- name: Fetch Config and Log Files
-  fetch:
-    dest: "error_files/{{inventory_hostname}}/"
+- name: Recreate directory
+  file:
+    path: "/tmp/error_files/{{inventory_hostname}}/"
+    state: absent
+
+- name: Recreate directory
+  file:
+    path: "/tmp/error_files/{{inventory_hostname}}/"
+    state: directory
+
+- name: Copy Config and Log Files to tmp dir
+  copy:
```

Comment:
I like this! Can you put in the description of this PR what the contents of the archive look like? 
I've seen with the copy sometimes it'll create dirs which may be a little annoying.